doc/c_sdk_resource_information.md

```diff
@@ -0,0 +1,38 @@
+# C-SDK Resource Information
+
+Detailed here are the captured the C SDK Resource constraints for each protocol.
+
+All test are executed on an ubuntu 18.04.1 image with GCC version 7.3.0 using C-SDK version 1.2.12.
+
+## On Disk Binary Information
+
+This is the size of the binary built with the c sdk for each protocol.  The appication setups the SDK and sends 1 telemetry message.
+
+| Protocol                | Size In Bytes
```

Comment:
>Size In Bytes [](start = 28, length = 13)

Would be nice to break this down for the following configs:
- SDK with everything in.
- With everything minus uploadToBlob
- With everything minus uploadToBlob & Logging

And include the cmake flag that builds each above in the footnotes.